src/flask_cognito_lib/decorators.py

```diff
@@ -140,6 +193,11 @@ def cognito_logout(fn):
     @wraps(fn)
     def wrapper(*args, **kwargs):
         with app.app_context():
+            # Revoke the refresh token if it exists
+            if refresh_token := session.get("refresh_token"):
```

Comment:
walrus operator :eyes:

Reply:
AFAIK it is supported from Python 3.8 which is also a requirement for the lib, so should be ok.